sdk_contrib/testbed/src/test/java/io/opentelemetry/sdk/contrib/trace/testbed/clientserver/Client.java

```diff
@@ -40,10 +42,10 @@ public void send() throws InterruptedException {
     span.setAttribute("component", "example-client");
 
     try (Scope ignored = tracer.withSpan(span)) {
-      tracer
+      OpenTelemetry.getPropagators()
           .getHttpTextFormat()
           .inject(
-              span.getContext(),
+              Context.current(),
```

Comment:
Is there any way to get the context of a particular `span`?

Reply:
@Oberon00 The `Context`? No, there's no way to get it (observe `Span` could be attached to multiple `Context` objects too ;) )